cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SchemaIndexTest.scala

```diff
@@ -272,6 +319,32 @@ class SchemaIndexTest extends DocumentingTestBase with QueryStatisticsTestSuppor
     )
   }
 
+  @Test def use_index_with_starts_with_composite() {
+    executePreparationQueries(List("CREATE INDEX ON :Person(firstname, surname)"))
+
+    executePreparationQueries {
+      val a = (0 to 100).map { i => "CREATE (:Person)" }.toList
+      val b = (0 to 300).map { i => s"CREATE (:Person {firstname: '$i', surname: '${-i}'})" }.toList
+      a ++ b
+    }
+
+    sampleAllIndexesAndWait()
+
+    profileQuery(
+      title = "Prefix search using `STARTS WITH` (composite index)",
+      text =
+        "The `STARTS WITH` predicate on `person.firstname` in the following query will use the `Person(firstname,surname)` index, if it exists. " +
+        "Any (non-existence check) predicate on `person.surname` will be rewritten as existence check with a filter. " +
+        "However, if the predicate on `person.firstname` is a equality check then a `STARTS WITH` on `person.surname` would also use the index (without rewrites).",
```

Comment:
Maybe you should refer to the first section with the general rules here as well so people understand the logic behind this last sentence.